Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/MotionController/ControllerPoseSynchronizer.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.EventDatum.Input;
+using Microsoft.MixedReality.Toolkit.Internal.Extensions;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers;
+using Microsoft.MixedReality.Toolkit.SDK.Input;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.MotionController
+{
+    /// <summary>
+    /// Waits for a controller to be initialized, then synchronizes its transform position to a specified handedness.
+    /// </summary>
+    public class ControllerPoseSynchronizer : InputSystemGlobalListener, IMixedRealitySourcePoseHandler
```

Comment:
If we are agreed with this rename.  This should be done in the visualization PR, as that has already made changes to this (and the inspector) classes.
So please revert these and I'll rename in the visualizer PR

Reply:
I still need these changes for the pointer viz, which is why I decided to do it separately.

Reply:
Can we then move this file to another folder then, say handlers? (including it's inspector)

Reply:
Yes